docs/pull_request.md

```diff
@@ -0,0 +1,158 @@
+
+##### Avoid unrelated fixing other issue in a big PR
+
+Each PR should focus on a single task. If other issues may be fixed when working in the area of it, it's preferable to open a dedicated PR.
+
+It will have the advantage to be reviewed and merged faster.
+
+#### Bots
+
+Some bots can create PR, but they still have to be reviewed by the team
+
+##### Dependabot
+
+Dependabot is a tool which maintain all our external dependencies up to date. A dedicated PR is created for each new available release for one of our external dependency.Dependabot
+
+To review such PR, you have to
+ - **IMPORTANT** check the diff files (as always).
+ - Check the release note. Some existing bugs in Element project may be fixed by the upgrade
+ - Make sure that the CI is happy
+ - If the code does not compile (API break for instance), you have to checkout the branch and push new commits
+ - Do some smoke test, depending of the library which has been upgraded
+
+For some reason dependabot sometimes does not upgrade some dependencies. In this case, and when detected, the upgrade has to be done manually.
+
+##### Gradle wrapper
+
+`Update Gradle Wrapper` is a tool which can create PR tu upgrade our gradle.properties file.
+Review such PR is the same recipe than for PR from Dependabot
+
+##### Sync analytics plan
+
+This tools import any update in the analytics plan. See instruction in the PR itself to handle it.
+More info can be found in the file [analytics.md]
+
+## Reviewing PR
+
+### Who can review pull requests?
+
+As an open source project, every one can review each others PR. Of course an approval from internal developer is mandatory for a PR to be merged.
+But comment in PR from the community are always appreciated!
+
+### What to have in mind when reviewing a PR
+
+1. User experience: is the UX and UI correct? You will probably be the second person to test the new thing, the first one is the developer.
+2. Developer experience: does the code looks nice and decoupled? No big functions, etc.
+3. Code maintenance. A bit similar to point 2. Tricky code must be documented for instance
+4. Fork consideration. Is configuration of forks will be easy? Some documentation may help in some cases.
+
+### Rules
+
+#### Check the form
+
+##### PR description
+
+PR description should follow the PR template, and at least provide some context about the code change.
+
+##### File change
+
+1. Code should follow the guidelines
+2. Code should be formatted correctly
+3. XML attribute must be sorted
+4. New code is added at the correct location
+5. New classes are added to the correct location
+6. Naming is correct. Naming is really important, it's considered part of the documentation
+7. Architecture is followed. For instance, the logic is in the ViewModel and not in the Fragment
+8. There is at least one file for the changelog. Exception if the PR fixes something which has not been released yet.
```

Comment:
Maybe add a link here explaining how the `changelog` works, .misc .sdk .feature etc

Reply:
Yes I agree, as a general feedback, this file lacks lot of links to both internal and external locations.